### ISSUE TITLE
Deprecate Alert - background-color only without icon

### ISSUE THREAD

**Issue by humancompanion-usds** [department-of-veterans-affairs/vets-design-system-documentation#1759]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

Deprecate Alert - background-color only without icon is not WCAG compliant because it relies on color alone to identify and differentiate itself from other alerts. Thus we should no longer support it, mark it as deprecated in the Design System, and remove it as an option from va-alert.

## What I expected to happen

Alerts must have more than 1 visual cue that signals their level of importance. That can be color AND icon but not just one of those.

## Reproducing
https://design.va.gov/components/alert#examples---background-color-only

## Urgency
While not blocking work, use of this component is a WCAG violation and one we should quickly address.

### Acceptance criteria
* [x] Background-only alert (default, without icon) is marked as deprecated in guidance with details as to why
* [ ] Background-only alert (default, without icon) is removed from va-alert
* [x] Background-only alert (default, without icon) is removed from Sketch Current library
* [x] Background-only alert (default, without icon) is removed from Sketch MVP library

**Comment by caw310** [comment]
Will carry over into the next sprint.

**Comment by humancompanion-usds** [comment]
I need to add this to release notes once #908 is done.

**Comment by caw310** [comment]
@jamigibbs  will take care of Background-only alert (default, without icon) is removed from va-alert.
@babsdenney will take care of removing from Sketch library.

**Comment by jamigibbs** [comment]
> Background-only alert (default, without icon) is removed from va-alert

This will be completed as part of https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/908

**Comment by LillyBoot** [comment]
@babsdenney Background-only (default, without icon) has been removed from the MVP Sketch libraries

**Comment by jamigibbs** [comment]
Once https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/908 is merged, this issue will auto-close.
